### ISSUE TITLE
`dbt debug` requires the existence of `default` cluster to pass

### ISSUE THREAD

**Issue by chuck-alt-delete** [MaterializeInc/materialize#19318]
### What version of Materialize are you using?

v0.54.0 (4f02a15a4)

### What is the issue?

Version:
```bash
➜ dbt --version
Core:
  - installed: 1.5.0
  - latest:    1.5.0 - Up to date!

Plugins:
  - postgres:    1.4.6 - Not compatible!
  - materialize: 1.4.1 - Not compatible!
```

Here is a successful dbt run against a materialize environment with no default cluster:
```bash
➜ dbt run
21:34:30  Running with dbt=1.5.0
21:34:30  Found 5 models, 2 tests, 0 snapshots, 0 analyses, 324 macros, 0 operations, 0 seed files, 2 sources, 0 exposures, 0 metrics, 0 groups
21:34:30  
21:34:34  Concurrency: 1 threads (target='dev')
21:34:34  
21:34:34  1 of 5 START sql source model public.auction_house ............................. [RUN]
21:34:37  1 of 5 OK created sql source model public.auction_house ........................ [CREATE SOURCE in 2.96s]
21:34:37  2 of 5 START sql view model public.on_time_bids ................................ [RUN]
21:34:38  2 of 5 OK created sql view model public.on_time_bids ........................... [CREATE VIEW in 1.43s]
21:34:38  3 of 5 START sql view model public.avg_bids .................................... [RUN]
21:34:40  3 of 5 OK created sql view model public.avg_bids ............................... [CREATE VIEW in 1.73s]
21:34:40  4 of 5 START sql view model public.highest_bid_per_auction ..................... [RUN]
21:34:41  4 of 5 OK created sql view model public.highest_bid_per_auction ................ [CREATE VIEW in 1.42s]
21:34:41  5 of 5 START sql materializedview model public.winning_bids .................... [RUN]
21:34:43  5 of 5 OK created sql materializedview model public.winning_bids ............... [CREATE MATERIALIZED VIEW in 1.83s]
21:34:43  
21:34:43  Finished running 1 source model, 3 view models, 1 materializedview model in 0 hours 0 minutes and 13.11 seconds (13.11s).
21:34:43  
21:34:43  Completed successfully
21:34:43  
21:34:43  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
```

However, `dbt debug` fails the connection test:
```bash
➜ dbt debug
21:34:50  Running with dbt=1.5.0
21:34:50  dbt version: 1.5.0
21:34:50  python version: 3.11.2
...
21:34:50  Configuration:
21:34:51    profiles.yml file [OK found and valid]
21:34:51    dbt_project.yml file [OK found and valid]
21:34:51  Required dependencies:
21:34:51   - git [OK found]

21:34:51  Connection:
...
21:34:52  1 check failed:
21:34:52  dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  unknown cluster 'default'
```

Related:
- https://github.com/MaterializeInc/materialize/pull/17197

**Comment by chuck-alt-delete** [comment]
Also tests:

```
22:12:45  Completed with 2 errors and 0 warnings:
22:12:45  
22:12:45  Database Error in test not_null_on_time_bids_bid_id (models/auction_house.yml)
22:12:45    unknown cluster 'default'
22:12:45    compiled Code at target/run/auction_house/models/auction_house.yml/not_null_on_time_bids_bid_id.sql
22:12:45  
22:12:45  Database Error in test unique_on_time_bids_bid_id (models/auction_house.yml)
22:12:45    unknown cluster 'default'
22:12:45    compiled Code at target/run/auction_house/models/auction_house.yml/unique_on_time_bids_bid_id.sql
```

**Comment by morsapaes** [comment]
This is a known issue after we released `dbt-materialize` v1.4.1 (see #19042). We need to pass the custom `cluster` from `profiles.yml` as an argument in the connection string (now that we can). Fix incoming!

**Comment by morsapaes** [comment]
This has been fixed in Materialize [v0.61](https://materialize.com/docs/releases/v0.61/). Health check queries no longer required a valid active cluster.
